.travis.yml

```diff
@@ -17,7 +26,7 @@ matrix:
      rust: beta
 before_install: |
   if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-    brew install unixodbc
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install unixodbc
```

Comment:
Workaround for https://travis-ci.community/t/syntax-error-unexpected-keyword-rescue-expecting-keyword-end-in-homebrew/5623

Reply:
That would make a great comment for the travis file itself.

Reply:
Done